opcuaSource/src/test/java/io/vantiq/extsrc/opcua/uaOperations/Connection.java

```diff
@@ -766,8 +780,34 @@ public void makeConnection(boolean runAsync,
             pubServers = Arrays.asList(Utils.OPC_INPROCESS_SERVER);
         }
 
+        int workingServers = 0;
+        int successfulConnections = 0;
         for (String discEP : pubServers) {
             log.info("Attempting connection to public server: " + discEP);
+            try {
+                URI opcuri = new URI(discEP);
+                // We cannot just use the URL since we don't have handling for the opc.tcp scheme.
+                // We'll just do a basic socket connection as that's where most of the issues w/r/t availability are.
+                String host = opcuri.getHost();
+                int    port = opcuri.getPort();
```

Comment:
Is this intentional spacing for `int port`? I don't have an opinion either way, just checking.

Reply:
no -- will fix